ADP Data Plugin Documentation (#19)
DEFRA/adp-documentation: docs/Getting-Started/onboarding-a-delivery-programme.md

## Creating a Delivery Programme

Once you have navigated to the 'ADP Data' page you will be presented with the 'Delivery Programmes' option. 
![ADP Data](../images/adp-data.png)
By clicking 'View' you will have the ability to view existing Delivery Programmes and add new ones if you have the admin permissions. 
![View Delivery Programmes](../images/adp-view-delivery-programme.png)

## Entering Delivery Programme information

You can start entering Delivery Programme information by clicking the 'Add Delivery Programme' button.
![Add Delivery Programme](../images/adp-create-delivery-programme.png)
You will be presented with various fields; some are optional. For example, the 'Finance Code', 'Website', and 'Alias' are not required, and you can add them later if you wish.

> The Arms Length Body (ALB) that was created for your programme should appear in the Arms Length Body dropdown, and you should be able to select it accordingly. The programme managers' dropdown should also be pre-populated, and you are able to select more than one manager.

Reviewer: ```suggestion
If the Arms Length Body (ALB) for your programme has already been created it will appear in the Arms Length Body dropdown and you will be able to select it accordingly. The programme managers' dropdown should also be pre-populated, and you are able to select more than one manager.
```
